lte/gateway/docker/docker-compose.yaml

```diff
@@ -359,3 +359,53 @@ services:
       timeout: "4s"
       retries: 3
     command: /var/opt/magma/bin/envoy_controller
+
+  # for docker compose v1: https://github.com/docker/compose/issues/8351#issuecomment-1085811366
```

Comment:
It might be nice to mention this in https://github.com/magma/magma/issues/13938 or to create an issue about replacing this with the wait flag once we upgrade to v2?